spring-cloud-schema-registry-server/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM java:8-alpine
+
+ARG JAR_FILE
+
+ADD target/${JAR_FILE} spring-cloud-skipper-server.jar
```

Comment:
I guess you've copied the Dockerfile from the skipper project but forgot to change the jar names. 
Replace the `spring-cloud-skipper-server.jar` by `spring-cloud-schema-registry.jar` instead.